src/main/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -679,13 +682,15 @@ private void calculateTargetCoverageMetrics() {
                     zeroCoverageTargets++;
                     highQualityCoverageHistogramArray[0] += c.interval.length();
                     targetBases[0] += c.interval.length();
+                    minDepth = 0;
```

Comment:
could you explain this one?

Reply:
I think I understand: there's no coverage `!c.hasCoverage()`....so minDepth=0

Reply:
Yes, in the case where any target interval has no coverage, the minDepth must be 0